nixos/modules/services/web-apps/hedgedoc.nix

```diff
@@ -1031,7 +1031,8 @@ in
       '';
       serviceConfig = {
         WorkingDirectory = cfg.workDir;
-        StateDirectory = [ cfg.workDir cfg.configuration.uploadsPath ];
+        StateDirectory = [ (builtins.replaceStrings [ "/var/lib/"  ] [ "" ] cfg.workDir) ];
```

Comment:
Why does this not use `removePrefix`? Now, if `/var/lib/` occurs twice in the string, that also gets removed.

Reply:
I wasn't aware of it's existence, thanks for the hint.